.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
                 command: cargo build --workspace
             - run:
                 name: Test Trin workspace
-                command: cargo test --workspace
+                command: cargo test --workspace -- --nocapture
```

Comment:
Are you good with dropping this change, after the Discord conversation?

Reply:
Tbh, I don't mind dropping it. But I'm a bit confused. `"oh yeah, I never figured out how to get peertest to capture on success and display on failure."` ... I'm interpreting this as meaning that, yes basically peertest doesn't capture all of the output as a normal test would? Shouldn't this be an argument for leaving the flag in? So that the missing output will be displayed in ci. Or am I misunderstanding here

Reply:
Leaving this change in for now, but happy to remove it in a future pr if i'm misunderstanding the situation